$OpenBSD: patch-src_main_c,v 1.11 2011/07/12 08:12:29 dcoppa Exp $

Raise event on incomplete QuickTime file, and mark it for retry
(fix apple.com/trailers) - upstream svn revision r2152

--- src/main.c.orig	Mon Jun 27 16:59:45 2011
+++ src/main.c	Mon Jul 11 10:19:19 2011
@@ -534,6 +534,7 @@ gint play_iter(GtkTreeIter * playiter, gint restart_se
     idledata->width = width;
     idledata->height = height;
 
+	idledata->retry_on_full_cache = FALSE;
     idledata->cachepercent = -1.0;
     g_strlcpy(idledata->info, uri, 1024);
     set_media_info(idledata);
